cumulusci.yml

```diff
@@ -449,16 +466,16 @@ flows:
     config_unmanaged:
         steps:
             1:
-                task: enable_course_connections
```

Comment:
Really curious about the reason why we are reordering these tasks.

Reply:
This was deliberate as the install script should run before any other scripts to address dependencies or potential dependencies that we expose.